utilities/test_suite/HOST/runTests.py

```diff
@@ -51,19 +51,19 @@ def get_log_file_list(preserveOutput):
 
 # Functionality group finder
 def func_group_finder(case_number):
-    if case_number < 5 or case_number == 13 or case_number == 36 or case_number == 31 or case_number == 45:
```

Comment:
If this func_group_finder() is repeated between HIP and HOST, please put it in common.py

Reply:
Done